src/Selenium2Driver.php

```diff
@@ -999,216 +933,107 @@ public function maximizeWindow($name = null)
-        if ($element->attribute('value') === $value) {
-            $element->click();
-
-            return;
-        }
-
-        $name = $element->attribute('name');
-
-        if (!$name) {
-            throw new DriverException(sprintf('The radio button does not have the value "%s"', $value));
-        }
-
-        $formId = $element->attribute('form');
-
-        try {
-            if (null !== $formId) {
-                $xpath = <<<'XPATH'
-//form[@id=%1$s]//input[@type="radio" and not(@form) and @name=%2$s and @value = %3$s]
-|
-//input[@type="radio" and @form=%1$s and @name=%2$s and @value = %3$s]
-XPATH;
-
-                $xpath = sprintf(
-                    $xpath,
-                    $this->xpathEscaper->escapeLiteral($formId),
-                    $this->xpathEscaper->escapeLiteral($name),
-                    $this->xpathEscaper->escapeLiteral($value)
-                );
-                $input = $this->wdSession->element('xpath', $xpath);
-            } else {
-                $xpath = sprintf(
-                    './ancestor::form//input[@type="radio" and not(@form) and @name=%s and @value = %s]',
-                    $this->xpathEscaper->escapeLiteral($name),
-                    $this->xpathEscaper->escapeLiteral($value)
-                );
-                $input = $element->element('xpath', $xpath);
-            }
-        } catch (NoSuchElement $e) {
-            $message = sprintf('The radio group "%s" does not have an option "%s"', $name, $value);
+        if ('input' !== strtolower($element->getTagName()) || $type !== strtolower($element->getAttribute('type'))) {
+            $message = 'Impossible to %s the element with XPath "%s" as it is not a %s input';
 
-            throw new DriverException($message, 0, $e);
+            throw new DriverException(sprintf($message, $action, $xpath, $type));
         }
-
-        $input->click();
     }
 
-    /**
-     * @param Element $element
-     * @param string  $value
-     * @param bool    $multiple
-     */
-    private function selectOptionOnElement(Element $element, $value, $multiple = false)
+    private function uploadFile($path)
     {
-        $escapedValue = $this->xpathEscaper->escapeLiteral($value);
-        // The value of an option is the normalized version of its text when it has no value attribute
-        $optionQuery = sprintf('.//option[@value = %s or (not(@value) and normalize-space(.) = %s)]', $escapedValue, $escapedValue);
-        $option = $element->element('xpath', $optionQuery);
-
-        if ($multiple || !$element->attribute('multiple')) {
-            if (!$option->selected()) {
-                $option->click();
-            }
-
-            return;
-        }
-
-        // Deselect all options before selecting the new one
-        $this->deselectAllOptions($element);
-        $option->click();
+        throw new \RuntimeException('Not yet supported');
```

Comment:
is this even called anywhere ? If no, remove the private method

Reply:
this method removed.